inclusive-design/idrc
# feat: scaffold about pages, implement menu system and breadcrumbs

## Description

Adds _About / Overview_ and _About / Philosophy_ pages. Implements progressively-enhanced menu system and breadcrumbs.

## Steps to test

1. Visit deploy preview.
2. Test menus on mobile and desktop with Javascript enabled and disabled.
3. Review About pages.

**Expected behavior:** Menu design and behaviour meets expectations. Breadcrumb design and behaviour meets expectations. About pages match designs.

## Additional information

Note: About pages sidebar navigation will be resolved an a separate pull request. The news page has not been migrated under the About pages yet, just into the menu.

## Related issues

- Resolves #53.
- Resolves #54.
- Resolves #59.
